include/usvfs.h

```diff
@@ -154,6 +154,30 @@ DLLEXPORT VOID WINAPI BlacklistExecutable(LPWSTR executableName);
  */
 DLLEXPORT VOID WINAPI ClearExecutableBlacklist();
 
+/**
+ * adds a file suffix to a list to skip during file linking
+ * .txt and some_file.txt are both valid file suffixes,
+ * not to be confused with file extensions
+ * @param fileSuffix  a valid file suffix
+ */
+DLLEXPORT VOID WINAPI AddSkipFileSuffix(LPWSTR fileSuffix);
```

Comment:
Although most other functions are not prefixed, I think it would be a good idea to prefix with `usvfs` similar to `usvfsConnectVFS` and `usvfsCreateVFS`.